node-definitions/juniper/vsrx/README.md

```diff
@@ -0,0 +1,9 @@
+# Juniper vSRX Node Definitions
+
+This directory contains the following node definitions:
+
+* `jnpr-vsrx.yaml` - Juniper vSRX 3.0 Virtual Firewall
+
+## Caveats
+
+vSRX 3.0 support is considered **EXPERIMENTAL** within CML.
```

Comment:
Any hint to here people could get the QCOW2 from?

Reply:
README.md updated